azurerm/internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -239,6 +269,12 @@ func resourceArmAppConfigurationUpdate(d *schema.ResourceData, meta interface{})
 		Tags: tags.Expand(d.Get("tags").(map[string]interface{})),
 	}
 
+	if _, ok := d.GetOk("identity"); ok {
+		parameters.Identity = expandAppConfigurationIdentity(d.Get("identity").([]interface{}))
+	} else {
+		parameters.Identity = &appconfiguration.ResourceIdentity{Type: "None"}
+	}
```

Comment:
same here - this can become:

```suggestion
	if d.HasChange("identity") {
		parameters.Identity = expandAppConfigurationIdentity(d.Get("identity").([]interface{}))
	}
```

which'll handle both the if and the else here - since the expand handles this